src/ui/actions/timeline.ts

```diff
@@ -202,8 +217,9 @@ export function setTimelineState(state: Partial<TimelineState>): SetTimelineStat
 export function setTimelineToTime(time: number | null, updateGraphics = true): UIThunkAction {
   return async ({ dispatch, getState }) => {
     dispatch(setTimelineState({ hoverTime: time }));
+    const isTrimming = getModal(getState()) === "trimming";
```

Comment:
Maybe pull out `state` here as a local so we don't call `getState` potentially three times?